test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1333,7 +1333,7 @@ def expected_seeded_manifest(self, model_database=None, quote_model=False):
                     'checksum': self._checksum_file(seed_path),
                     'unrendered_config': unrendered_seed_config,
                 },
-                'test.test.not_null_model_id': {
+                'test.test.not_null_model_id.8ab00aacfe': {
```

Comment:
Are we going to have to manually write down these hashes for all these tests in the future? Not sure how this test works.

Reply:
Probably for the time being we will.  IMO this add-a-hash solution needs to be replaced with either:
- something human readable
- something better instrumented so we don't have to manage these in tests/etc.

Either way, I think it'll be a larger project to re-work the way node-naming works so this feels stop-gap-ish.

Reply:
Maybe we should file a tech debt ticket and reference this PR so we don't forget.

Reply:
[Doneski](https://github.com/fishtown-analytics/dbt/issues/3348)